nixpkgs/BUILD.bazel

```diff
@@ -59,6 +62,7 @@ bzl_library(
     deps = [
         ":bazel_tools",
         ":nixpkgs",
+        "//toolchains/go:go.bzl",
```

Comment:
I'm a bit puzzled. This is only the backwards compatibility file, right? I.e. the one that reexports. Shouldn't there be a dependency on `@rules_nixpkgs_go` somewhere?

Reply:
No, this points to the new module (the old file would be `//nixpkgs/toolchains:go.bzl`). The target is only used for docs generation, and indeed appears a bit misplaced. For some reason all the other language specifics had been in `nixpkgs` and only `go` was separate. I would clean this up step by step, for example when separating out tests in #194. This PR is already huge.

Reply:
Thanks for clarifying. Yes, happy to defer this to a follow-up PR.